packages/connect/src/utils/assets.ts

```diff
@@ -9,10 +9,10 @@ if (global && typeof global.fetch !== 'function') {
     global.fetch = fetch;
 }
 
-export const httpRequest = (url: string, _type: string): any => {
+export const httpRequest = (url: string, type: string): any => {
```

Comment:
🤔 checking FW authenticity is not the only place were this util is used with type param, right? How does this change affect the other places? 

https://github.com/trezor/trezor-suite/blob/fix-fw-authenticity-desktop/packages/connect/src/api/firmware/getBinary.ts/#L33

https://github.com/trezor/trezor-suite/blob/fix-fw-authenticity-desktop/packages/connect/src/data/DataManager.ts/#L59